init.sh

```diff
@@ -0,0 +1,10 @@
+if [! -d .oh-my-zsh]
+then 
+	sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
+fi
+ln -s .dotfiles/.vimrc $HOME/.vimrc
+ln -s .dotfiles/.zshrc $HOME/.zshrc
+rm $ZSH_CUSTOM/aliases.zsh
```

Comment:
I'm referring to line number 7, when you remove $ZSH_CUSTOM/aliases.zsh. You should probably check for its existence before you remove it. Something like:
`if [ -f $ZSH_CUSTOM/aliases.zsh ]; then; rm $ZSH_CUSTOM/aliases.zsh;`